packages/plugins/color-picker/admin/src/components/tests/ColorPickerInput.test.tsx

```diff
@@ -0,0 +1,66 @@
+import { DesignSystemProvider } from '@strapi/design-system';
+import { render as renderRTL } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+
+import { ColorPickerInput } from '../ColorPickerInput';
+
+const render = () => ({
+  ...renderRTL(
+    <ColorPickerInput
+      name="color"
+      value=""
+      onChange={jest.fn()}
+      attribute={{
+        customField: 'plugin::color-picker.color',
+        pluginOptions: { i18n: { localized: true } },
+        type: 'string',
+      }}
+      intlLabel={{ id: 'color-picker', defaultMessage: 'color-picker' }}
+    />,
+    {
+      wrapper: ({ children }) => {
+        const locale = 'en';
+        return (
+          <IntlProvider locale={locale} messages={{}} textComponent="span">
+            <DesignSystemProvider locale={locale}>{children}</DesignSystemProvider>
+          </IntlProvider>
+        );
+      },
+    }
+  ),
+  user: userEvent.setup(),
+});
+
+describe('<ColorPickerInput />', () => {
+  /**
+   * We do this because –
+   * https://github.com/facebook/jest/issues/12670
+   */
+  beforeAll(() => {
+    jest.setTimeout(30000);
+  });
+
+  /**
+   * Reset timeout to what is expected
+   */
+  afterAll(() => {
+    jest.setTimeout(5000);
+  });
+
+  it('renders and matches the snapshot', () => {
+    const { container } = render();
+
+    expect(container).toMatchSnapshot();
```

Comment:
Is it the way the diff is displayed or is the snapshot really empty?

Reply:
think it's just the diff